lib/sign_in_page.dart

```diff
@@ -9,29 +11,136 @@ class SignInPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Column(
+      body: Stack(
         children: [
-          TextField(
-            controller: emailController,
-            decoration: InputDecoration(
-              labelText: "Email",
-            ),
+          Column(
+            children: [
+              Padding(
+                padding: EdgeInsets.only(top: 50.0),
+                child: Container(
+                  height: 150,
+                  width: 150,
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.all(Radius.circular(30.0)),
+                    color: Colors.black12,
+                    image: DecorationImage(
+                      image: AssetImage('assets/images/Logo.png'),
+                    ),
+                  ),
+                ),
+              ),
+              Container(
+                padding: EdgeInsets.only(top: 50.0, left: 20.0),
+                alignment: Alignment.topLeft,
+                child: Text(
+                  "LOGIN",
+                  style: TextStyle(
+                    fontSize: 36.0,
+                    fontFamily: 'Cornerstone',
+                    color: Color(0xFF113654),
+                  ),
+                ),
+              ),
+              Padding(
+                padding: EdgeInsets.only(top: 5.0, left: 20.0, right: 20.0),
+                child: TextField(
+                  controller: emailController,
+                  decoration: InputDecoration(
+                    labelText: "Email",
+                    filled: true,
+                    fillColor: Colors.black12,
+                  ),
+                ),
+              ),
+              Padding(
+                padding: EdgeInsets.only(top: 15.0, left: 20.0, right: 20.0),
+                child: TextField(
+                  controller: passwordController,
```

Comment:
Same as the email text field, we want the system's keyboard to know that this is a password field so keyboard autocomplete won't try to remember the user's passwords. To do this, you'd just add `keyboardType: TextInputType.visiblePassword,` as one of the properties.

In addition to that, we also want to obscure the password field text when the user is entering in their password. That's also another easy fix, just add `obscureText: true,`

Note that you might need to do a soft app restart rather than just a hot reload to see the keyboard difference